src/redux/slices/userSlice.ts

```diff
@@ -1,37 +1,32 @@
 import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import * as wallet from "../../services/wallets/wallet";
 import * as session from "../../services/session";
-import { Profile } from "../../utilities/types";
+import { DSNPUserId } from "@dsnp/sdk/dist/types/core/identifiers";
 
 interface UserState {
-  profile?: Profile;
+  id?: DSNPUserId;
   walletType: wallet.WalletType;
 }
 
 const initialState: UserState = {
-  profile: session.hasSession() ? session.loadSession()?.profile : undefined,
```

Comment:
does this affect the session storage?

Reply:
It does. Good catch. I added it back using id instead.